aGHQ.qmd

```diff
@@ -622,11 +642,11 @@ end
 fit!(com05fe, β₀);
 ```
 
-The IRLS algorithm has converged in 4 iterations to essentially the same deviance as the general optimizer achieved after around 500 function evaluations.
+The IRLS algorithm has converged in 4 iterations to essentially the same deviance as the general optimizer achieved after more than 150 function evaluations.
```

Comment:
I feel like we should be grabbing this value from the code and rounding to something stable instead of manually updating.

[noblock]

Reply:
The difference here is that PRIMA declares convergence in fewer iterations than does NLopt.  Nevertheless, I take your point and will see about squirreling away the number of iterations to convergence.